sympy/printing/latex.py

```diff
@@ -2199,39 +2199,39 @@ def translate(s):
         return s
 
 
-def latex(expr, **settings):
+def latex(expr, fold_frac_powers=False, fold_func_brackets=False, fold_short_frac=None, inv_trig_style="abbreviated", itex=False, ln_notation=False, long_frac_ratio=None, mat_delim="[", mat_str=None, mode="plain", mul_symbol=None, order=None, symbol_names={}):
     r"""Convert the given expression to LaTeX string representation.
 
     Parameters
     ==========
-    fold_frac_powers : boolean, optional
+    fold_frac_powers : boolean, optional (default = False)
         Emit ``^{p/q}`` instead of ``^{\frac{p}{q}}`` for fractional powers.
-    fold_func_brackets : boolean, optional
+    fold_func_brackets : boolean, optional (default = False)
         Fold function brackets where applicable.
-    fold_short_frac : boolean, optional
+    fold_short_frac : boolean, optional (default = None)
```

Comment:
This change is not necessary. It is already included in the description and also in the method signature.